README.md

```diff
@@ -1,48 +1,37 @@
 # Welcome to Aethos
 This is the Aethos Operator documentation - a guide which provides instructions on setting up and running the Aethos Operator using either Docker or a binary executable.
 
-## We're live on Testnet
-Aethos Testnet Phase Two is underway on Holesky 
+## Aethos Testnet
+Aethos testnet phase three is live on Holesky!
 - Service Manager Proxy [```0xdE93E0dA148e1919bb7f33cd8847F96e45791210```](https://holesky.etherscan.io/address/0xdE93E0dA148e1919bb7f33cd8847F96e45791210)
-- Aggregator is running on ```holesky.task.aethos.network:50051```
+- Aggregator service is running on ```holesky.task.aethos.network:50051```
 
-## Prerequisites to Running an Operator
-* **Must be registered with Eigenlayer on Holesky** Account registered with Eigenlayer as an operator (see [here](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation)).
+## Run an Aethos Operator
+
+### Hardware Requirements
+* **CPU:** 4 Cores
+* **Memory:** 16 GB
+* **Storage:** 100 GB SSD
+
+### Prerequisites
+* **Registered with Eigenlayer on Holesky** Registration key registered with Eigenlayer (see [here](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation)).
+* **Defined Aethos AVS Signer Key** Signer Key used for signing tasks (see [here](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/key-management/intro#eigenlayer)).
+* **Aethos team enabled registration key:** Aethos testnet is permissioned. Your registration key must be on the allowlist of the Aethos ServiceManger contract. Please reach out to us if this has not already been complete.
 * **Holesky node (full/archive):** You can point to your local instance or to an RPC provider.
-* **Enabled operator:** Aethos testnet is permissioned. Your operator address must be on the allowlist on our ServiceManger contract. Please reach out to us if this has not already been complete.
 
-## Networking
-* **Expose 9010 port for `18.118.236.196` :** exposing this port enables inbound tasks
-* **Expose 9090 port for `18.118.236.196`:** exposing this port enable metrics collection
-  
-## Configuration
+### Expose the following IP/Ports
+* **`18.118.236.196`:** exposing this port enables inbound tasks
+* **`18.118.236.196`:** exposing this port enable metrics collection
+
+### Configuration
 The Aethos Operator supports configuration via command-line interface (CLI) arguments or a config.yaml file.
 If both methods are used, CLI arguments will take precedence over configurations specified in the config.yaml file.
 
-### Configuration Preferences:
+Preferences
 * Docker: It's preferred to use CLI arguments.
 * Binary: It's preferred to use the config.yaml file.
 
-### Configuration Options
-Below are the available configuration options for the Aethos Operator:
-
-```sh
-GLOBAL OPTIONS:
```

Comment:
can we add back latest configuration options for each command `operator register` and `operator start`

Reply:
Is It worth the space ? I think you can run --help

Reply:
would recommend just adding that comment somewhere to see the configuration options

Reply:
i do agree that it might not be worth the space